docs/visualization/plotting-1d-data.ipynb

```diff
@@ -39,7 +39,7 @@
    "source": [
     "d = sc.Dataset()\n",
     "N = 50\n",
-    "d.coords['tof'] = sc.Variable(['tof'], values=np.arange(N).astype(np.float64),\n",
+    "d.coords['tof'] = sc.Variable(['tof'], values=np.arange(N, dtype=np.float64),\n",
```

Comment:
Very minor point, we could also move the dtype specification to the `Variable` itself. I'm not sure if that has any major benefit other than showing that we have dtype control at the point of variable creation.